src/tsickle_compiler_host.ts

```diff
@@ -1,12 +1,12 @@
 import * as path from 'path';
-import {SourceMapConsumer, SourceMapGenerator} from 'source-map';
+import {SourceMapGenerator} from 'source-map';
 import * as ts from 'typescript';
 
 import {convertDecorators} from './decorator-annotator';
 import {processES5} from './es5processor';
 import {ModulesManifest} from './modules_manifest';
+import * as sm from './source_map_utils';
```

Comment:
The Go rule around short names is that the more local and contextual the name (e.g. the iterator for a "for" loop), the more likely you are to know the name in context (e.g. "i").  For a global name like this that appears across the file, a longer name like `sourceMapUtils` is more appropriate.

(The JS/TS style rule is to just always use long names.)